pyuploadcare/resources/file_group.py

```diff
@@ -164,4 +164,7 @@ def store(self):
         if self.is_stored:
             return
 
+        # TODO: here we can inject working with files API to be complaint with
+        # existed code
```

Comment:
In progress?

Reply:
Technically, that is solved [here](https://github.com/uploadcare/pyuploadcare/pull/217), but I also delete TODO for that branch too